packages/search-in-workspace/src/common/search-in-workspace-interface.ts

```diff
@@ -90,16 +111,7 @@ export namespace SearchInWorkspaceResult {
         if (a.fileUri !== b.fileUri) {
             return a.fileUri < b.fileUri ? -1 : 1;
         }
-
-        if (a.line !== b.line) {
-            return a.line - b.line;
-        }
-
-        if (a.character !== b.character) {
-            return a.character - b.character;
-        }
-
-        return a.length - b.length;
+        return 0;
```

Comment:
why do the "line" and "character" not matter any more?

Reply:
Because a result contains all matches of a file